draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -176,7 +176,10 @@ $row-height: 60px;
 .expanded {
   // We want to have NO background on Heart theme (when CSS variables are defined), but keep stone backgrond on Zen. Need to do this using 0 opacity.
   // We also only want this switch for the expanded state. Non-expanded hovering will resort to stone background, as seen in .hasHoverState
-  --dt-hover-background-heart: rgba(var(#{$kz-var-id-color-white}), 0);
+  --dt-hover-background-heart: rgba(
+    var(#{$kz-var-id-color-white-rgb-params}),
```

Comment:
Not suggesting a change, but I'm curiouso: is the `#{...}` interpolationo syntax needed here? We don't seem to be using it in other places in this commit.